docs/4.x/translations.md

```diff
@@ -127,6 +127,7 @@ Follow these guidelines when creating your own translation keys:
 
 1. **Reuse!** If a core plugin contains a translation you can use, use that instead. If there's a translation you want to use but can't because it's in the wrong case, don't use functions like `lcfirst` and `ucfirst` as it won't work in all languages and rather create a new translation key.
 2. **Use numbered placeholders** if more than one is required in your text.
+3. HTML like breaks and links should not be in the translation. Use placeholders like `%1$s` instead and pass the HTML as arguments when you translate. 
```

Comment:
We actually can use HTML in translations. Weblate should even ensure that the same html tags are used in the translation. Using placeholders might actually even make it harder to translate, as a translator doesn't know what a placeholder is filled with. For complexe tags (with attributes) I would say we should avoid. But simple line breaks or even formatting tags like `<i>`, `<strong>` or similar would imho be fine to use.

Reply:
Have we done that in the past? does it mean if we change the HTML then translations will be invalidated?

Reply:
Yes and yes.
```
➜  ~/public_html/matomo git:(ui-tests-bionic) ✗ rg -F "<strong" **/en.json
plugins/PrivacyManager/lang/en.json
51:        "RawDataNeverRemoved": "visits and actions raw data are <strong>never<\/strong> deleted.",
53:        "ReportsNeverRemoved": "aggregated reports are <strong>never<\/strong> deleted.",

plugins/CorePluginsAdmin/lang/en.json
56:        "SuccessfullyActicated": "You have successfully activated <strong>%s<\/strong>.",
➜  ~/public_html/matomo git:(ui-tests-bionic) ✗ rg -F "<br" **/en.json
plugins/Referrers/lang/en.json
67:        "SocialsReportDocumentation": "This report shows which social networks led visitors to your website.<br />By clicking on a row in the table, you can see from which social network pages visitors came to your website.",

plugins/PrivacyManager/lang/en.json
32:        "AnonymizeUserIdHelp": "When you enable this option, the User ID will be replaced by a pseudonym to avoid directly storing and displaying personally identifiable information such as an email address. In technical terms: given your User ID, Matomo will process the User ID pseudonym using a salted hash function.<br\/><br\/><em>Note: replacing with a pseudonym is not the same as anonymisation. In GDPR terms: the User ID pseudonym still counts as personal data. The original User ID could still be identified if certain additional information is available (which only Matomo and your data processor has access to).<\/em>",

plugins/Login/lang/en.json
5:        "ContactAdmin": "Possible reason: your host may have disabled the mail() function. <br \/>Please contact your Matomo administrator.",
[...] (and quite a few more
```